source/index.txt

```diff
@@ -32,12 +28,11 @@ You can add the driver to your application to work with MongoDB
 in JavaScript. Download it using `npm <https://www.npmjs.com/>`__
 or set up a runnable project by following our Quick Start guide.
 
-
 Quick Start
 -----------
 
 Learn how to establish a connection to MongoDB Atlas and begin
-working with data in the :doc:`Quick Start </quick-start>` section.
+working with data in the step-by-step :doc:`Quick Start </quick-start>`.
```

Comment:
Thanks for pointing that out! Since this PR isn't initially going to be backported, I think it might be better to leave alone and address somewhere else.